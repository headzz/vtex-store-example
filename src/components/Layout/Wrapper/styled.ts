import styled from 'styled-components'

export const WrapperStyled = styled.div`
  height: 100%;
  width: 100%;
`
