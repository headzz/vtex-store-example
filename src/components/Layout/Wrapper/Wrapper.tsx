import { WrapperStyled } from './styled'

/**
 * @component
 * @name Wrapper
 * @description Defines the container for the layout.
 * @namespace components components/Layout/Wrapper
 * @param {element} children Component(s) that may appear inside the layout.
 *
 */

const Wrapper: React.FC = ({ children }) => {
  return <WrapperStyled>{children}</WrapperStyled>
}

export default Wrapper
