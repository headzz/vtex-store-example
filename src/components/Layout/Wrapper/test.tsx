import { render } from '@testing-library/react'

import Wrapper from './Wrapper'

describe('<Wrapper />', () => {
  it('should render component that wraps everything on page', () => {
    const { container } = render(<Wrapper />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
