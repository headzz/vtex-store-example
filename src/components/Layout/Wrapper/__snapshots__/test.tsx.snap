// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `<Wrapper /> should render component that wraps everything on page 1`
] = `
.c0 {
  height: 100%;
  width: 100%;
}

<div
  class="c0"
/>
`
